### PR TITLE
improve ordinal plural docs

### PR DESCRIPTION
**Pull request by xdumaine**
## Motivation:

I had a really hard time understanding the existing docs for getting ordinal plurals to work. After much trial and error, I figured it out, and I believe these docs make it much clearer.

#### Checklist (for documentation change)

- [x] only relevant documentation part is changed (make a diff before you submit the PR)
- [x] motivation/reason is provided
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/i18next/.github/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by adrai** [comment]
Thank you.
Can you please revert " -> '

**Comment by xdumaine** [comment]
Oh yep, sorry about that, prettier ran automatically.

File: translation-function/plurals.md [hunk 1]
```diff
@@ -95,27 +95,29 @@ _Or try_ [_translation-check_](https://github.com/locize/translation-check)_, it
...
 keys
 
 ```javascript
-// i.e. italian
+// i.e. english
 {
-  "key_one": "singular", // cardinal form
-  "key_other": "other", // cardinal form
-  "key_many": "many" // ordinal form
+  "key_one": "{{position}}st place", // 1st, 21st, 31st
```
Comment (adrai) [inline translation-function/plurals.md:106]: I think instead position, this is count

Comment (xdumaine) [reply translation-function/plurals.md]: oof, yep, thank you for the good review.

**Comment by adrai** [comment]
thank you for your contribution

**Pull request merged**